sbnobj/Common/Calibration/classes_def.xml

```diff
@@ -0,0 +1,6 @@
+<lcgdict>
+  <class name="sbn::TrackInfo" />
```

Comment:
Include ClassVersion here (and in the classes below). Will allow for version updating/reading old products if these persist for whatever reason.

Reply:
Gotcha -- how do you do that exactly? I am not familiar.

Reply:
See here: https://github.com/SBNSoftware/sbnobj/blob/867ce19432565d983144faa2e0ff41cec59d09a6/sbnobj/Common/Reco/classes_def.xml#L2

You just need to add in the ClassVersion part, and when you compile it should automatically calculate and add the checksum.

Reply:
Fixed.